### ISSUE TITLE
Is there ignoring a log operation in SCL?

### ISSUE THREAD

**Issue by TFboys-lzz** [HobbitLong/SupContrast#31]
Hi HobbitLong:
    Thank you for releasing code. Here, i have a question. Is there ignoring a log operation in 89 line of loss.py.

```
# compute log_prob
        exp_logits = torch.exp(logits) * logits_mask
        log_prob = logits - torch.log(exp_logits.sum(1, keepdim=True))

```

**Comment by HobbitLong** [comment]
Hi, it's because the `log` and `exp` for the numerator get cancelled out.
